riptide-stream/src/test/java/org/zalando/riptide/stream/EnforceCoverageTest.java

```diff
@@ -37,14 +37,14 @@
 
     @Test
     public void shouldUseStreamsConstructor() {
-        new Streams();
+        assertNotNull(new Streams());
     }
 
     @Test
     public void shouldUseStreamConverterConstructor() {
-        Streams.streamConverter();
-        Streams.streamConverter(null);
-        Streams.streamConverter(null, null);
+        streamConverter();
```

Comment:
I still strongly believe that those should be covered by real tests. We only cover useless constructors in here, i.e. hidden constructors from static factory classes.